app/controllers/creators_controller.rb

```diff
@@ -1,2 +1,31 @@
 class CreatorsController < ApplicationController
+
+  def index
+    if @current_user.creators != []
+      @creators = @current_user.creators
+    else
+      flash.now[:notice] = "Try following some people first!"
+    end
+  end
+
+  def follow
+    creator = Creator.find_or_create(params)
+    if creator.nil?
+      flash[:error] = "Failed to follow"
+    else
+      if !@current_user.creators.include?(creator)
```

Comment:
According to the Ruby Style Guide, it is the preferred way to use `unless` instead of negative `if` conditionals.
Ex:
```ruby
unless @current_user.creators.include?(creator)
```
You can find the Ruby Style Guide [here](https://github.com/bbatsov/ruby-style-guide)